src/app/lib/views/title_bar.js

```diff
@@ -71,12 +71,7 @@
         },
 
         minimize: function () {
-            var that = this.nativeWindow;
-            if (AdvSettings.get('minimizeToTray')) {
-                minimizeToTray();
-            } else {
-                that.minimize();
-            }
+            this.nativeWindow.minimize();
```

Comment:
instead of using a minimize custom function, I intercept the on('minimize') event, that's nwjs native event. Without the modification, the only way to minimize to tray is to click on the minimize button, not any other way (native OS shortcut, clicking on icon, etc). Tray works on Ubuntu, there's no reason it wouldnt work on gnome